includes/rest-api/class-wp-job-manager-models-settings.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Declaration of our Settings Model
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Models_Settings
+ */
+class WP_Job_Manager_Models_Settings extends WPJM_REST_Model_Declaration_Settings
```

Comment:
I think it will, under two conditions (which might not hold, but I'd rather they did):

1. Plugins will want the settings endpoint to return a flat object
2. Classes override `MT_Model_Declaration_Settings ::get_settings` and return a correctly shaped array.